src/utils/selectors/filterBanners.js

```diff
@@ -0,0 +1,13 @@
+export const filterBanners = (banners) => {
```

Comment:
The `filter` word is not correct, you are doing a mapping. Filtering is excluding and map is transform.